source/docs/transportation/alt-fuel-stations-v1/nearest.html.md.erb

```diff
@@ -430,8 +986,9 @@ The individual fuel station records are returned in order of ascending distance
 <pre>GET <a href="/api/alt-fuel-stations/v1/nearest.csv?api_key=DEMO_KEY&amp;latitude=39.743078&amp;longitude=-105.152278&amp;fuel_type=ELEC&amp;limit=1">/api/alt-fuel-stations/v1/nearest.csv?api_key=DEMO_KEY&amp;latitude=39.743078&amp;longitude=-105.152278&amp;fuel_type=ELEC&amp;limit=1</a></pre>
```

Comment:
```suggestion
<pre>GET <a href="/api/alt-fuel-stations/v1/nearest.csv?api_key=DEMO_KEY&amp;latitude=39.743078&amp;longitude=-105.152278&amp;fuel_type=ELEC&amp;limit=2">/api/alt-fuel-stations/v1/nearest.csv?api_key=DEMO_KEY&amp;latitude=39.743078&amp;longitude=-105.152278&amp;fuel_type=ELEC&amp;limit=2</a></pre>
```
Based on the example output, it looks like this should be `limit=2` instead for `CSV Output Format`. It looks like the same change is needed for `GeoJSON Output Format` at https://github.com/NREL/developer.nrel.gov/blob/related-stations-docs/source/docs/transportation/alt-fuel-stations-v1/nearest.html.md.erb#L566.

Reply:
Thanks for catching that! Fixed in a009620568a